### PR TITLE
Cleanup incomplete CLI command 

### PR DESCRIPTION
**Pull request by WilliamXieCrypto**
Following this issue https://github.com/PeggyJV/gravity-bridge/issues/366

Open a PR to cleanup the incomplete&half-implemented CLI command.

### CODE REVIEW

File: orchestrator/gorc/src/commands/tests.rs [hunk 1]
```diff
@@ -1,26 +0,0 @@
...
-/// Run tests against configured chains
-#[derive(Command, Debug, Parser)]
-pub enum TestsCmd {
-    Runner(Runner),
-}
-
-impl Runnable for TestsCmd {
-    /// Start the application.
-    fn run(&self) {
-        // Your code goes here
```
Comment (WilliamXieCrypto) [inline orchestrator/gorc/src/commands/tests.rs:12]: unimplemented.

Comment (WilliamXieCrypto) [inline orchestrator/gorc/src/commands/tests.rs:12]: unimplemented.

File: orchestrator/gorc/src/commands/tx.rs [hunk 2]
```diff
@@ -1,24 +0,0 @@
...
-    Cosmos(cosmos::Cosmos),
-
-    #[clap(subcommand)]
-    Eth(eth::Eth),
-}
-
-impl Runnable for TxCmd {
-    /// Start the application.
-    fn run(&self) {
-        // Your code goes here
```
Comment (WilliamXieCrypto) [inline orchestrator/gorc/src/commands/tx.rs:22]: unimplemented.

Comment (WilliamXieCrypto) [inline orchestrator/gorc/src/commands/tx.rs:22]: unimplemented.

**Review by EricBolten — APPROVED** [review]
LGTM

**Pull request merged**